zynthian_gui.py

```diff
@@ -721,6 +721,9 @@ def show_screen(self, screen=None, hmode=SCREEN_HMODE_ADD):
 			else:
 				return
 
+		if screen not in ("bank", "preset", "option"):
```

Comment:
This function gets called a lot - one almost every screen navigation.

Reply:
If pre-selection is toggled as favourite then this still fails, i.e. preset is restored.